cygnus-ngsi-ld/src/main/java/com/telefonica/iot/cygnus/utils/NGSIConstants.java

```diff
@@ -51,7 +51,7 @@ private NGSIConstants() {
     public static final String ATTR_VALUE          = "attrValue";
     public static final String ATTR_MD             = "attrMd";
     public static final String ATTR_MD_FILE        = "attrMdFile";
-    
+    public static final String AUTOGENERATED_ATTR_TYPE           = "_attr_internal_type";
```

Comment:
Is your fork updated? This line was added some weeks ago, it's weird that GitHub marks it as difference. Maybe you should update your upstream and update your branches.

Reply:
If I am correct the latest commits amend that